### DISCUSSION TITLE
Last feeding card: show time since end of last feeding, not start?

### DISCUSSION THREAD

**Discussion by abog854** [babybuddy/babybuddy#781] [Ideas]
Hi,

Would it make more sense to show, in the dashboard "last feeding" card, the time since the last feeding ended, not when it started?

Me and my partner use the time since last feeding / since last change a lot, and we want to know the time since the baby finished feeding, not since it started (it's more relevant w.r.t. baby hungriness levels).
For feedings that last 30 minutes it makes a meaningful difference.

I'm happy to send a PR, but I don't know how the maintainers feel about it.

**Comment by cdubz** [comment]
I try to base reasoning as much as possible around research and/or convention. For breastfeding everything I've ever read that deals with breastfeeding timing says to measure from start to start. I can't say there is a clear or obvious reason for that but I do think it qualifies as the convention. Moving this to a discussion for now.
